.github/workflows/ci.yml

```diff
@@ -47,15 +47,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
```

Comment:
Just realized - there is no need for this.

Reply:
Opened #3402

Reply:
@RobPasMue Ok. We will remove it soon. Thank you.